Build fails when compiling with GCC 6.2

Build fails when compiling with GCC 6.2.

GCC gives this output:

```
src/LineShape.cpp:13:71: error: 'sqrt' was not declared in this scope

src/LineShape.cpp:17:46: error: 'atan2' was not declared in this scope

src/Terminal.cpp:111:42: error: call of overloaded 'abs(float&)' is ambiguous
/[...]/include/stdlib.h:338:15: note: candidate: int abs(int)
/[...]/include/c++/6.2.0/cstdlib:185:3: note: candidate: __int128 std::abs(__int128)
/[...]/include/c++/6.2.0/cstdlib:180:3: note: candidate: long long int std::abs(long long int)
/[...]/include/c++/6.2.0/cstdlib:172:3: note: candidate: long int std::abs(long int)
```

--- a/src/LineShape.cpp
+++ b/src/LineShape.cpp
@@ -4,6 +4,8 @@
 
 #include "LineShape.hpp"
 
+#include <cmath>
+
 LineShape::LineShape(sf::Vector2f &point1, sf::Vector2f &point2, float thickness, sf::Color color) :
     Point1(point1),
     Point2(point2),

--- a/src/Terminal.cpp
+++ b/src/Terminal.cpp
@@ -5,6 +5,7 @@
 #include "Terminal.hpp"
 #include "Util.hpp"
 
+#include <cmath>
 #include <iostream>
 
 using namespace std;

